shared/src/main/scala/translit/Russian.scala

```diff
@@ -0,0 +1,115 @@
+package translit
+
+import translit.Helpers._
+
+object Russian extends Language {
+  val uniGrams = Map(
+    'a' -> 'а',
+    'b' -> 'б',
+    'v' -> 'в',
+    'g' -> 'г',
+    'd' -> 'д',
+    'e' -> 'е',
+    'z' -> 'з',
+    'i' -> 'и',
+    'j' -> 'й',
+    'k' -> 'к',
+    'l' -> 'л',
+    'm' -> 'м',
+    'n' -> 'н',
+    'o' -> 'о',
+    'p' -> 'п',
+    'r' -> 'р',
+    's' -> 'с',
+    't' -> 'т',
+    'u' -> 'у',
+    'f' -> 'ф',
+    'x' -> 'х',
+    'h' -> 'х',
+    'c' -> 'ц',
+    'w' -> 'щ',
+    '#' -> 'ъ',
+    'y' -> 'ы'
+  )
+
+  val biGrams = Map(
+    "jo" -> 'ё',
```

Comment:
Are there any spelling conventions in Russian with regards to j vs y? I think we should restrict ourselves to only one spelling. I would propose "yo" and "ya" which we also use for our Ukrainian transliteration.

Reply:
It depends on personal preferences and I've seen people use both. I was basing the rules on `translit.net`. But yeah, I will change it to only `y`